envs/monkey_zoo/blackbox/utils/credential_config_generator/credential_config_generator.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+import copy
+import json
+import os
+from pathlib import Path
+
+import nanoid as nanoid
+
+parser = argparse.ArgumentParser(
+    prog="Credential Config Generator",
+    description="Small script to add usernames and " "passwords to Infection Monkey configuration",
+)
+
+parser.add_argument(
+    "-u",
+    "--userlist-path",
+    type=str,
+    required=True,
+    help="Path to userlist file. Userfile should contain one username per line.",
+)
```

Comment:
This is kind of confusing.
```suggestion
parser.add_argument(
    "-u",
    "--usernamelist-path",
    type=str,
    required=True,
    help="Path to file with list of usernames. File should contain one username per line.",
)
```